### ISSUE TITLE
Request: Add mixin for #sizedImgDiv

### ISSUE THREAD

**Issue by kristfal** [PolymerElements/iron-image#79]
### Description

A use case for `iron-image` can be to add a box shadow inset in order to render a gradient across the image, such as:

```
-webkit-box-shadow: inset 0px -29px 55px -18px rgba(0,0,0,0.65);
box-shadow: inset 0px -29px 55px -18px rgba(0,0,0,0.65);
```

Setting this style on `iron-image` will not render as `#sizedImgDiv` is rendered on top of the `iron-image` tags.

### Expected outcome

Should be able to use box-shadow insets with `iron-image`

### Actual outcome

Box shadow insets is not rendering because they need to be applied to `#sizedImgDiv`

### Proposed solution

Add a mixin to `#sizedImgDiv`

Let me know if you want a PR.

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [x] IE 10

**Comment by bicknellr** [comment]
If you'd like to submit a PR, feel free; this doesn't seem contentious or breaking.

**Comment by kristfal** [comment]
@bicknellr See #83. Also, tests are failing for Firefox – there seems to be an issue with the CI.

**Comment by bicknellr** [comment]
Yeah, we've been having some issues with Firefox and the test framework lately. Thanks for the PR. After considering this again, I think this might be outside the scope of iron-image given that img also behaves this way. [Here's a demo.](http://jsbin.com/bihimel/1/edit)
